### PR TITLE
Turn in assignment

### PR DESCRIPTION
**Pull request by aolavessen**

### CODE REVIEW

**Comment by rtablada** [comment]
screen shot 2016-05-20 at 8 29 44 am

On all sizes
- Don't show the hamburger
- Center the icon on the picture using [transform](https://rtablada.gitbooks.io/spring-2016-fee/content/week-1/day-3/transform.html)
- A bit more space between the icons and the text
- No underline on navbar links
- Use a sans serif font to match the design

On phone sizes
- Give some breathing room on the edges of the photos

screen shot 2016-05-20 at 8 33 16 am

File: styles/_main.scss [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+.main__article {
+   display: flex;
+   justify-content: flex-end;
+   margin: auto;
+
+  //  padding-left: 5vw
+}
+
+.pic-and-author {
+  flex-basis: calc(60% - 2rem);
```
Comment (rtablada) [inline styles/_main.scss]: If you don't have margin, this `-2rem` is going to mess with things centering up on the screen

**Comment by rtablada** [comment]
@aolavessen can you set the body font-family to some sort of sans serif?

**Comment by aolavessen** [comment]
Tis done.

**Comment by rtablada** [comment]
screen shot 2016-05-26 at 12 12 45 pm
@aolavessen on mobile sizes, there's still no whitespace around the text and the edge...?

**Comment by rtablada** [comment]
@aolavessen that's something you might be able to get with some padding on your container instead of putting the padding on the text itself

**Pull request merged**